models/Headers.yaml

```diff
@@ -0,0 +1,21 @@
+openapi: 3.1.0
+info:
+  title: DCSA Headers
+  description: |
+    Common header components for DCSA APIs.
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+  contact:
+    name: Digital Container Shipping Association (DCSA)
+    url: 'https://dcsa.org'
+    email: info@dcsa.org
+  version: 3.0.0
```

Comment:
Is this smart to keep in a Models file? If yes - I suggest that you rename the file to `Headers-OVS.yaml` or something similar

Reply:
why though? I do see common headers that can be present so there is value in having a common file.

Reply:
That is also a solution :-)